docs/en/docs/tutorial/request-form-models.md

```diff
@@ -36,6 +36,14 @@ You can verify it in the docs UI at `/docs`:
 <img src="/img/tutorial/request-form-models/image01.png">
 </div>
 
+## Including `File` Fields { #including-file-fields }
+
+You can also include **file fields** inside a form model by using `UploadFile` or `bytes` as field types and annotating them with `File`.
+
+If there is at least one `File` field in the form model, **FastAPI** will automatically set the request content media type to `multipart/form-data` for this path operation.
```

Comment:
Should we add here a note to indicate from which version onwards, this is the case? Something like 
```suggestion
If there is at least one `File` field in the form model, **FastAPI** (from v.0.121.3 onwards) will automatically set the request content media type to `multipart/form-data` for this path operation.
```